analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -91,10 +91,8 @@ class GoMod(
         val projectDir = definitionFile.parentFile
 
         stashDirectories(projectDir.resolve("vendor")).use {
-            val fullGraph = getDependencyGraph(projectDir)
-            val projectId = fullGraph.projectId()
-
-            val graph = fullGraph.subgraph(getUsedPackages(fullGraph, projectDir, projectId))
+            val graph = getDependencyGraph(projectDir)
```

Comment:
Commit message:
* Typo in "repor~~i~~ting"
* I don't get the second sentence, do you mean something like "...returning unused dependencies and leaving their removal to the caller"?